.github/ISSUE_TEMPLATE/issue_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Issue report
```

Comment:
Issue/Bug/Crash/Unexpected Result Report

Reply:
changed to something similar!